internal/interfaces/types.go

```diff
@@ -28,3 +32,24 @@ type ServiceClass struct {
 	Priority int                 `yaml:"priority"`
 	Data     []ServiceClassEntry `yaml:"data"`
 }
+
+// PrometheusConfig holds complete Prometheus client configuration including TLS settings
+type PrometheusConfig struct {
+	// BaseURL is the Prometheus server URL
+	BaseURL string `json:"baseURL"`
+
+	// TLS configuration fields
+	EnableTLS          bool   `json:"enableTLS,omitempty"`
+	InsecureSkipVerify bool   `json:"insecureSkipVerify,omitempty"`
```

Comment:
Do we need two booleans for EnableTLS and InsecureSkipVerify? Can we use just one boolean and infer the other?

Reply:
Wed ont need it anymore, removed